dockerfiles/ink-waterfall-ci/Dockerfile

```diff
@@ -25,6 +25,19 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN	set -eux; \
 	apt-get -y update && \
+	apt-get install -y --no-install-recommends \
+# `firefox` is needed to simulate interactions with the `canvas-ui`
+	firefox && \
```

Comment:
Not against firefox at all, just to make a point.
We use chromium at substrate CI:
https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-linux/Dockerfile#L26

Reply:
Okay good to know, but the library which I'm using supports only geckodriver, which is tied to firefox.

Reply:
cool